setup.rst

```diff
@@ -24,12 +24,12 @@ Getting Set Up
 Version Control Setup
 ---------------------
 
-CPython is developed using `Mercurial <http://hg-scm.org/>`_. The Mercurial
-command line program is named ``hg``; this is also used to refer to Mercurial
-itself. Mercurial is easily available for common Unix systems by way of the
-standard package manager; under Windows, you might want to use the
-`TortoiseHg <http://tortoisehg.org/>`_ graphical client, but the build system
-still prefers :file:`hg.exe` to be on your PATH.
+CPython is developed using `Git <https://git-scm.com/>`_. The Git command line
+program is named ``git``; this is also used to refer to Git itself. Git is
+easily available for common Unix systems by way of the standard package manager;
+under Windows, you might want to use the `TortoiseGit <http://tortoisegit.org/>`_
```

Comment:
It might be worth also linking to https://desktop.github.com/ as well at TortoiseGit.